### PR TITLE
incubator/cassandra v0.4.2 - Templating for liveness/readiness probes

### PR DESCRIPTION
**Pull request by Jaskaranbir**
**What this PR does / why we need it**:

Adds customizable liveness and readiness probe parameters.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A

**Special notes for your reviewer**:
None

### CODE REVIEW

File: incubator/cassandra/templates/cassandra-statefulset.yaml [hunk 1]
```diff
@@ -103,13 +103,30 @@ spec:
         livenessProbe:
           exec:
             command: [ "/bin/sh", "-c", "nodetool status" ]
+          {{- if not .Values.livenessProbe }}
```
Comment (davidkarlsen) [inline incubator/cassandra/templates/cassandra-statefulset.yaml]: is this really needed when you have the default section lower down?

Comment (Jaskaranbir) [reply incubator/cassandra/templates/cassandra-statefulset.yaml]: True that, thanks for the suggestion.

File: incubator/cassandra/templates/cassandra-statefulset.yaml [hunk 2]
```diff
@@ -103,13 +103,19 @@ spec:
         livenessProbe:
           exec:
             command: [ "/bin/sh", "-c", "nodetool status" ]
-          initialDelaySeconds: 90
-          periodSeconds: 30
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds | default 90 }}
```
Comment (davidkarlsen) [inline incubator/cassandra/templates/cassandra-statefulset.yaml]: are the defaults needed as long as the values are defined in the values.yaml? they act as defaults?

Comment (Jaskaranbir) [reply incubator/cassandra/templates/cassandra-statefulset.yaml]: Ah, totally missed what you meant. Still learning Helm; should be fixed now.

**Pull request merged**